app/javascript/packs/initializers/initializeSettings.js

```diff
@@ -0,0 +1,65 @@
+import { timestampToLocalDateTime } from '@utilities/localDateTime';
+
+export class CopyOrgSecret {
```

Comment:
We don't really use JS classes much especially when they aren't encapsulating any data - for classes like this that only contain `static` methods, top-level functions are fine. For namespacing/code organization, moving groups of functions to another module and importing them works

Reply:
Got it, changed to modules.